pkg/profiler/cpu/maps.go

```diff
@@ -959,6 +960,9 @@ func (m *bpfMaps) setUnwindTableForMapping(buf *profiler.EfficientBuffer, pid in
 	}
 	buildID, err := buildid.FromELF(ef)
 	if err != nil {
+		if errors.Is(err, errTextSectionNotFound) {
```

Comment:
`buildid.FromELF` never actually return the `errTextSectionNotFound` so we will never go into this if. We need to move the error to the buildid package and return it here: https://github.com/parca-dev/parca-agent/blob/cefb78dc9c5582a57b2bbe62dd9fae3a0b20033b/pkg/buildid/elf.go#L66

Then we can check whether we're getting that error in this if with `if errors.Is(err, buildid.ErrTextSectionNotFound)`